fe/fe-core/src/main/java/com/starrocks/sql/StatementPlanner.java

```diff
@@ -65,7 +65,10 @@ public static ExecPlan plan(StatementBase stmt, ConnectContext session,
                 Analyzer.analyze(stmt, session);
             }
 
-            PrivilegeChecker.check(stmt, session);
+            // For table just crated by CTAS statement, we ignore the check of 'INSERT' privilege on it.
```

Comment:
Here is the main process of Planner, and it is suitable to add business logic here. Can it be moved into PrivilegeChecker::visitInsertStatement?

Reply:
Sure, sounds better, i will change it